### PR TITLE
Updated whats new and version number

### PR DESCRIPTION
**Pull request by kchawla-pi**

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -1,3 +1,93 @@
...
+
+**Nilearn Python2 support is being removed in the near future.
+Users with a Python2 environment will be warned at their first Nilearn import.**
+
+Improvements
+------------
+
+Displays created by plotting functions can now add a scale bar
+ to indicate the size in mm or cm (see :ref:`plotting`),
+ contributed by Oscar Esteban
```
Comment (GaelVaroquaux) [inline doc/whats_new.rst]: Is this valid restructured text? I guess that circle will tell us.

**Comment by GaelVaroquaux** [comment]
Waiting for CI to run, and then merge and releasing.

**Comment by kchawla-pi** [comment]
Forgot to update the website what's new.


On Oct 16, 2018 21:41, "Gael Varoquaux" <notifications@github.com> wrote:

Waiting for CI to run, and then merge and releasing.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/nilearn/nilearn/pull/1818#issuecomment-430372604>, or mute
the thread

**Comment by GaelVaroquaux** [comment]
I was thinking that we could wait for Circle to run, and then I could do it on my computer while doing the release.

**Comment by GaelVaroquaux** [comment]
Pushing now will reset Circle, and I want Circle to run to check the rst.

**Comment by GaelVaroquaux** [comment]
Build failed. I don't know why.

I'm sorry, Kshitij, release will happen tomorrow afternoon, after I give my presentation.

**Comment by kchawla-pi** [comment]
That's fine Gael, its only small stuff now. We will release when ready.

Not the error I expected... 
`/home/circleci/project/doc/connectivity/region_extraction.rst::image file not readable: connectivity/../auto_examples/03_connectivity/images/sphx_glr_plot_extract_regions_dictlearning_maps_001.png`
I will add the Whats New on the website.

**Comment by GaelVaroquaux** [comment]
The CircleCI failure is an http temporary failure. I am merging anyhow, to see if things run, and because it seems unrelated.

**Pull request merged**